jenkins/automation/alex.groovy

```diff
@@ -33,10 +33,14 @@ pipeline {
       stage('Install Dependencies') {
           steps {
               sh label: 'Installed Dependencies', script: '''
-                  yum install epel-release
-                  yum install nodejs
-                  npm install alex --global
+                  yum install epel-release -y
+                  curl -sL https://rpm.nodesource.com/setup_12.x | sudo bash -
+                  yum install nodejs -y
+                  npm install alex --global -y
+                  virtualenv alex --python=python2.7
+                  source ./alex/bin/activate
                   pip install GitPython==2.1.15
```

Comment:
use `pip3`

Reply:
Unfortunately, the script is written and testes in python2.7.

Reply:
Ok